src/device-registry/routes/api-v1.js

```diff
@@ -3659,4 +3659,80 @@ router.delete(
   airqloudController.delete
 );
 
+router.post(
+  "/airqlouds/center",
```

Comment:
New route for getting the center of an AirQloud. Most of the logic in here is just input validation.